code/modules/mining/shelters.dm

```diff
@@ -17,9 +17,8 @@
 	var/affected = get_affected_turfs(deploy_location, centered=TRUE)
 	for(var/turf/T in affected)
 		var/area/A = get_area(T)
-		if(is_type_in_typecache(A, banned_areas))
+		if(is_type_in_typecache(A, banned_areas) || T.z == 1)
```

Comment:
add /area/bluespace_locker to the banned_areas list ez

Reply:
No because that would mean I need to add every centcom level.

Reply:
would that stop abductor teleports?

Reply:
change https://github.com/yogstation13/Yogstation/blob/ab6808b76f907488f028f55d733ee6eb563c7868/code/modules/mining/equipment/survival_pod.dm#L46 to get_turf_global instead so it just works outside of the bluespace locker